menas/pom.xml

```diff
@@ -265,17 +265,6 @@
             <artifactId>guava</artifactId>
             <version>${guava.version}</version>
         </dependency>
-        <dependency>
```

Comment:
Why to remove the dependency altogether and not just the exclusion? The Atum is used within Menas, as the files bellow show. This way the dependency is transitive via _Utils_

Reply:
Thanks for the explanation, it is indeed cleaner as suggested. Done.